dash.el

```diff
@@ -71,6 +71,21 @@ special values."
          (setq it-index (1+ it-index))
          (!cdr ,l)))))
 
+(defmacro -doto (eval-initial-value &rest forms)
+  "Evals a form, then inserts that form as the 2nd argument to other forms.
```

Comment:
Use imperative (Eval, insert) instead of 3rd person form.  That is the emacs standard.  In the rest of the explanation, 3rd person is fine.

I think this description is OK.  Once you do this last small edit, I'm merging! :+1: